packages/slidev/node/options.ts

```diff
@@ -64,7 +64,7 @@ export async function resolveOptions(
     roots,
   }
 
-  return {
+  return globalThis.__SLIDEV_OPTIONS__ = {
```

Comment:
Do we have to use global variables to do this tho?

Reply:
Removed the usage of `globalThis` by extracting `presetIcons` from `client/uno.config.ts` to `node/setups/unocss.ts`